dockerfiles/docker-compose.yml

```diff
@@ -72,7 +75,8 @@ services:
       - ${PWD}/common/dockerfiles/entrypoints/common.sh:/usr/src/app/docker/common.sh
       - ${PWD}/common/dockerfiles/entrypoints/celery.sh:/usr/src/app/docker/celery.sh
       - ${PWD}/common/dockerfiles/scripts/wait_for_search.py:/usr/src/app/docker/scripts/wait_for_search.py
-      - ${PWD}/../readthedocs-ext:/usr/src/app/checkouts/readthedocs-ext
+      - ${PWD}/${RTDDEV_PATH_EXT:-../readthedocs-ext}:/usr/src/app/checkouts/readthedocs-ext
+      - ${PWD}/${RTDDEV_PATH_EXT_THEME:-../ext-theme}:/usr/src/app/checkouts/ext-theme
```

Comment:
I'm OK using `RTDDEV_PATH_EXT` for `readthedocs-ext` path here. However, I don't think that `ext-theme` is required in `celery`, `build` and `proxito`. I'd say that it may be required only in `web` probably, right?

Reply:
I assume it isn't needed at all, but is there a strong reason not to include it in these containers?

Reply:
If it's not required, I won't include it. Keeping consistency with the rest is good to me and also adding things that "do nothing" just adds confusions in my experience; considering that our docker compose is overwritten in different places.

Reply:
I'd disagree, I think the larger issue is running instances that don't all have the same code. In production, we don't have instances that only have select repositories, all of the instances have the same pile of code. Developing with only a partial checkout seems like a way to miss side effects from repositories

Reply:
Well, the same question is valid for production then: why are we installing things in the images that are not required? :smile:

Reply:
There is a big argument for consistency. Making our images/docker images have special checkout requirements would be a recipe for introducing non-deterministic bugs around repository interactions.

On the other side of this argument, what is the downside of having additional repository definitions? That we have extraneous code lying around? That seems very minor in comparison to potentially spending time hunting bugs or introducing downtime with uncaught bugs.